pkgs/by-name/ma/manticoresearch/package.nix

```diff
@@ -95,21 +95,33 @@ stdenv.mkDerivation (finalAttrs: {
     "-DWITH_MYSQL=1"
     "-DMYSQL_INCLUDE_DIR=${mariadb-connector-c.dev}/include/mariadb"
     "-DMYSQL_LIB=${mariadb-connector-c.out}/lib/mariadb/libmysqlclient.a"
+    "-DCONFDIR=${placeholder "out"}/etc"
+    "-DLOGDIR=/var/lib/manticoresearch/log"
+    "-DRUNDIR=/var/run/manticoresearch"
   ];
 
+  postFixup = ''
+    mkdir -p $out/lib/systemd/system
+    cp ${finalAttrs.src}/dist/deb/manticore.service.in $out/lib/systemd/system/manticore.service
+    substituteInPlace $out/lib/systemd/system/manticore.service \
```

Comment:
bummer that this is only installed when building a deb or rpm https://github.com/search?q=repo%3Amanticoresoftware%2Fmanticoresearch%20manticore.service.in&type=code